Connection ID sequence number space starts at 0 for all paths (#439)
quicwg/multipath: draft-ietf-quic-multipath.md

issue a Connection ID for at least one unused Path ID, as long as it remains
compatible with the peer's Maximum Path ID limit.

When issuing path-specific connection IDs, an endpoint associates a sequence number
as specified in {{Section 5.1.1 of QUIC-TRANSPORT}}. Each Path ID has its own
connection ID sequence number space whose initial value is 0. On non-initial paths
(i.e., Path ID different from 0), the initial path connection ID is provided by a

> PATH_NEW_CONNECTION_ID frame whose sequence number value is 0.

Reviewer: ```suggestion
PATH_NEW_CONNECTION_ID frame with a sequence number value of 0.
```
